docs/en/setup/backend/backend-upgrade.md

```diff
@@ -0,0 +1,26 @@
+# Backend upgrade
+First and most important thing is, SkyWalking agent caching the key-value of the metadata, i.e., the service id and service name, the serice instance id and service instance name, the endpoint id and endpoint name. The backend will not working after you clear the metadata from elasticsearch database but not restart the agent.
```

Comment:
1. not work
2. from database, not just for elasticsearch
3. Unless restart the agent.